### PR TITLE
updating sample for beta2 api

### LINKED ISSUE #18657: Update sample_create_client.py for Beta 3 API
The [sample_create_client.py](https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/containerregistry/azure-containerregistry/samples/sample_create_client.py) feels like it fits to the shape of the original Beta 1 sample.  Please consider making the following updates:

- [x] Refer to `ContainerRepository` as something other than client [here](https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/containerregistry/azure-containerregistry/samples/sample_create_client.py#L40) and [here](https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/containerregistry/azure-containerregistry/samples/sample_create_client.py#L61)
- [x] Since `ContainerRepository` isn't a client, we shouldn't be able to create an instance of it directly as we do [here](https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/containerregistry/azure-containerregistry/samples/sample_create_client.py#L46), unless this is a Python-specific requirement.
- [x] In the proposed Beta3 API, we're not exposing functionality to list tags on a repository directly as we do [here](https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/containerregistry/azure-containerregistry/samples/sample_create_client.py#L65).
- [x] If you'd like to illustrate the use of `ContainerRepository` in this sample, consider also including an illustration of `ArtifactRegistry`, as it is another Helper type that makes service calls.

### PR DESCRIPTION
**Pull request by seankane-msft**
closes #18657

### CODE REVIEW

**Review by annelo-msft — APPROVED** [review]
LGTM - thanks!  Possibly worth getting Python dev eyes on, per your judgment.

File: sdk/containerregistry/azure-containerregistry/samples/async_samples/sample_create_client_async.py [hunk 1]
```diff
@@ -27,53 +27,40 @@
...
         # [START create_registry_client]
         from azure.containerregistry.aio import ContainerRegistryClient
         from azure.identity.aio import DefaultAzureCredential
+        account_url = os.environ["CONTAINERREGISTRY_ENDPOINT"]
 
-        client = ContainerRegistryClient(self.account_url, DefaultAzureCredential())
+        client = ContainerRegistryClient(account_url, DefaultAzureCredential())
         # [END create_registry_client]
 
-    async def create_repository_client(self):
```
Comment (swathipil) [inline sdk/containerregistry/azure-containerregistry/samples/async_samples/sample_create_client_async.py:41]: did you move the creating repository client sample elsewhere?

**Review by swathipil — APPROVED** [review]
one question but LGTM

File: sdk/containerregistry/azure-containerregistry/samples/async_samples/sample_create_client_async.py (hunk 1 above)
Comment (seankane-msft) [reply sdk/containerregistry/azure-containerregistry/samples/async_samples/sample_create_client_async.py:41]: This object isn't meant to be instantiated directly so I removed the sample

**Pull request merged**